### ISSUE TITLE
OrderList error

### ISSUE THREAD

**Issue by james7272** [primefaces/primereact#291]
The 'Responsive Cars'  OrderList on the web site documentation page throws this error if a reorder button is pressed without a row selected. I get the same error in my app when updating the list after a re-order operation.

```
react-dom.production.min.js:164 TypeError: Cannot read property 'getBoundingClientRect' of undefined
    at Function.value (DomHandler.js:283)
    at t.value (OrderList.js:283)
    at t.value (OrderList.js:291)
    at commitLifeCycles (react-dom.production.min.js:149)
    at t (react-dom.production.min.js:158)
    at w (react-dom.production.min.js:169)
    at E (react-dom.production.min.js:168)
    at batchedUpdates (react-dom.production.min.js:171)
    at J (react-dom.production.min.js:54)
    at Pe (react-dom.production.min.js:71)
```

**Comment by cagataycivici** [comment]
I'm on it.

**Comment by cagataycivici** [comment]
Fixed during #308
